pipelinedp4j/tests/com/google/privacy/differentialprivacy/pipelinedp4j/spark/SparkTableTest.kt

```diff
@@ -69,23 +68,23 @@ class SparkTableTest {
 
     @Test
     fun keys_returnKeys() {
-        val data = spark.createDataset(listOf(Tuple2("key", "value")), Encoders.tuple(Encoders.STRING(), Encoders.STRING()))
+        val data = sparkSession.spark.createDataset(listOf(Tuple2("key", "value")), Encoders.tuple(Encoders.STRING(), Encoders.STRING()))
         val sparkTable = SparkTable(data, Encoders.STRING(), Encoders.STRING())
         val result = sparkTable.keys("stageName")
         assertThat(result.data.collectAsList()).containsExactly("key")
     }
 
     @Test
     fun keys_returnsValues() {
```

Comment:
`values_returnsValues`

Reply:
I kept the same name as in BeamTableTests [keys_returnsValues](https://github.com/google/differential-privacy/blob/0a80e37c1d08feaae798f9ce54cf12c1781a5927/pipelinedp4j/tests/com/google/privacy/differentialprivacy/pipelinedp4j/beam/BeamTableTest.kt#L153)

Reply:
then its a typo, could you change it in both places? We name tests in google "{methodNameUnderTest}_{inputState}_{expectedResult}". methodNameUnderTest is `values`.

Reply:
done